molecule/setup_with_collections/templates/collections/sample_techproducts_configs/solrconfig.xml

```diff
@@ -35,7 +35,7 @@
        that you fully re-index after changing this setting as it can
        affect both how text is indexed and queried.
   -->
-  <luceneMatchVersion>8.0.0</luceneMatchVersion>
+  <luceneMatchVersion>{{ solr_version }}</luceneMatchVersion>
```

Comment:
I would prefer to create a new variable called `solr_lucene_match_version` and set `solr_version` to that variable as default value. I think that is common to have lucene match version under solr version used.

Reply:
It's an example.